DigitalLearningSolutions.Web.Tests/Controllers/Register/RegisterAtNewCentreControllerTests.cs

```diff
@@ -0,0 +1,319 @@
+    using DigitalLearningSolutions.Data.Exceptions;
+    using DigitalLearningSolutions.Data.Models.Register;
+    using DigitalLearningSolutions.Data.Models.User;
+    using DigitalLearningSolutions.Data.Services;
+    using DigitalLearningSolutions.Data.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.Controllers.Register;
+    using DigitalLearningSolutions.Web.Extensions;
+    using DigitalLearningSolutions.Web.Helpers;
+    using DigitalLearningSolutions.Web.Models;
+    using DigitalLearningSolutions.Web.Tests.ControllerHelpers;
+    using DigitalLearningSolutions.Web.Tests.TestHelpers;
+    using DigitalLearningSolutions.Web.ViewModels.Register;
+    using FakeItEasy;
+    using FluentAssertions;
+    using FluentAssertions.AspNetCore.Mvc;
+    using Microsoft.AspNetCore.Http;
+    using Microsoft.Extensions.Primitives;
+    using Microsoft.FeatureManagement;
+    using NUnit.Framework;
+
+    public class RegisterAtNewCentreControllerTests
+    {
+        private const string IpAddress = "1.1.1.1";
+        private const int SupervisorDelegateId = 1;
+        private ICentresDataService centresDataService = null!;
+        private RegisterAtNewCentreController controller = null!;
+        private IFeatureManager featureManager = null!;
+
+        private PromptsService promptsService = null!;
+        private IRegistrationService registrationService = null!;
+        private HttpRequest request = null!;
+        private ISupervisorDelegateService supervisorDelegateService = null!;
+        private IUserService userService = null!;
+
+        [SetUp]
+        public void Setup()
+        {
+            centresDataService = A.Fake<ICentresDataService>();
+            registrationService = A.Fake<IRegistrationService>();
+            userService = A.Fake<IUserService>();
+            promptsService = A.Fake<PromptsService>();
+            featureManager = A.Fake<IFeatureManager>();
+            supervisorDelegateService = A.Fake<ISupervisorDelegateService>();
+            request = A.Fake<HttpRequest>();
+
+            controller = new RegisterAtNewCentreController(
+                    centresDataService,
+                    featureManager,
+                    promptsService,
+                    registrationService,
+                    supervisorDelegateService,
+                    userService
+                )
+                .WithDefaultContext()
+                .WithMockRequestContext(request)
+                .WithMockServices()
+                .WithMockTempData()
+                .WithMockUser(true);
+        }
+
+        [Test]
+        public void PersonalInformationPost_with_invalid_emails_fails_validation()
+        {
+            // Given
+            const int centreId = 3;
+            controller.TempData.Set(new InternalDelegateRegistrationData());
+            var userAccount = UserTestHelper.GetDefaultUserAccount();
+            var delegateAccount = UserTestHelper.GetDefaultDelegateAccount();

```

Comment:
Do you need to use these methods? Can you just do `new UserAccount()` etc? If so, that would make it more obvious to readers that you're not relying on the user accounts having particular properties.